Showcase/Showcase/Pages/Tokens/Color/NamedColor/NamedColor+Border.swift

```diff
@@ -0,0 +1,79 @@
+//
+
+import OUDS
+import OUDSTokensSemantic
+
+// swiftlint:disable identifier_name
+
+extension NamedColor {
+
+    enum Border: String, CaseIterable {
+        case colorBorderDefault
+        case colorBorderDefaultOnBackgroundEmphasized
+        case colorBorderEmphasized
+        case colorBorderEmphasizedOnBackgroundEmphasized
+        case colorBorderFocus
+        // case colorBorderBrandPrimaryOnBackgroundEmphasized
+        // case colorBorderBrandPrimary
+        // case colorBorderBrandSecondary
+        // case colorBorderBrandTertiary
+        // case colorBorderStatusPositive
+        // case colorBorderStatusInfo
+        // case colorBorderStatusWarning
+        // case colorBorderStatusNegative
+        // case colorBorderStatusAccent
+        case colorBorderOnBackgroundBrandPrimary
+        // case colorBorderOnBackgroundBrandSecondary
+        // case colorBorderOnBackgroundBrandTertiary
+
+        func token(from theme: OUDSTheme) -> ColorSemanticToken {
+            switch self {
+            case .colorBorderDefault:
+                return theme.colorBorderDefault
+            case .colorBorderDefaultOnBackgroundEmphasized:
+                return theme.colorBorderDefaultOnBackgroundEmphasized
+            case .colorBorderEmphasized:
+                return theme.colorBorderEmphasized
+            case .colorBorderEmphasizedOnBackgroundEmphasized:
+                return theme.colorBorderEmphasizedOnBackgroundEmphasized
+            case .colorBorderFocus:
+                return theme.colorBorderFocus
+//            case .colorBorderBrandPrimaryOnBackgroundEmphasized:
+//                return theme.colorBorderBrandPrimaryOnBackgroundEmphasized
+//            case .colorBorderBrandPrimary:
+//                return theme.colorBorderBrandPrimary
+//            case .colorBorderBrandSecondary:
+//                return theme.colorBorderBrandSecondary
+//            case .colorBorderBrandTertiary:
+//                return theme.colorBorderBrandTertiary
+//            case .colorBorderStatusPositive:
+//                return theme.colorBorderStatusPositive
+//            case .colorBorderStatusInfo:
+//                return theme.colorBorderStatusInfo
+//            case .colorBorderStatusWarning:
+//                return theme.colorBorderStatusWarning
+//            case .colorBorderStatusNegative:
+//                return theme.colorBorderStatusNegative
+//            case .colorBorderStatusAccent:
+//                return theme.colorBorderStatusAccent
+            case .colorBorderOnBackgroundBrandPrimary:
+                return theme.colorBorderOnBackgroundBrandPrimary
+//            case .colorBorderOnBackgroundBrandSecondary:
+//                return theme.colorBorderOnBackgroundBrandSecondary
+//            case .colorBorderOnBackgroundBrandTertiary:
+//                return theme.colorBorderOnBackgroundBrandTertiary
+            }
+        }
+    }
+}
```

Comment:
Normalement tu dois avoir des warnings tels que :
- "Blanket Disable Command Violation: The disabled 'identifier_name' rule should be re-enabled before the end of the file (blanket_disable_command)"

Les warnings désactivés dans un fichier doivent être réactivés ensuite.

Il faut ajouter à la fin du fichier, après un saut de ligne, les lignes : 

```swift
// swiftlint:enable identifier_name
```